dokuwiki/lib/plugins/doodle/style.css

```diff
@@ -28,50 +28,55 @@ div.dokuwiki table.inline td.notokay {
   color: rgba(0, 0, 0, 0);
 }
 
-.doodle__results .title_row, .doodle__results .fields_row, .doodle__results .data_row, .doodle__results .results_row, .doodle__results .input_row {
+table.doodle__results tbody tr, table.doodle__results thead tr {
 	display: grid;
 	width: 100%;
 }
 
-.doodle__results .fields_row div, .doodle__results .results_row div {
+table.doodle__results .results_row td {
 	background-color: lightblue;
 	text-align: center;
 	padding: 0.25em;
 }
 
-.doodle__results .title_row .title_caption, .doodle__results .input_row .input_caption {
+table.doodle__results .title_row .title_caption, table.doodle__results .input_row .input_caption {
 	text-align: center;
 	background-color: #bbd;
 	width: 100%;
```

Comment:
```suggestion
background-color: #bbd;
```

<img width="1640" height="362" alt="Image" src="https://github.com/user-attachments/assets/6ef196a6-13a2-42ab-83ed-2877a395d77d" />

Reply:
I can't reproduce this behaviour, but happy to make that change.